### DISCUSSION TITLE
Is the version in the Ubuntu repository still the wrong one? (it says it is 2.4.25)

### DISCUSSION THREAD

**Discussion by bladderrunner** [abraunegg/onedrive#2535] [Q&A] [answered]
In the installation guide here, https://github.com/abraunegg/onedrive/blob/master/docs/INSTALL.md

by Ubuntu 23.04 it says not to use the standard repository version is it is on 2.4.23

However, when I look in the Ubuntu software manager on my machine is shows onedrive-cli is available and is version 2.4.25.

I am not certain that is the same app as this one. It names Hideo Aoyama as the maker.

I'm something of a novice so would prefer to use that rather than follow all the steps to use the OpenSuse version which is recommended here. I wonder if that advice is now out of date if Ubuntu have updated their version?

Thanks for any advice!

**Comment by abraunegg** [comment, marked as answer]
@bladderrunner 
Please refer to: https://packages.ubuntu.com/search?keywords=onedrive

23.04 contains client version 2.4.23-1

So the documentation is 100% correct - do not use the Ubuntu repositories.

What you are seeing is `onedrive-cli` which is a 'snap' - https://snapcraft.io/onedrive-cli

I do not support the 'snap' - if this works for you - great, if you have issues with it, unfortunately you are on your own.

**Reply by bladderrunner** [reply]
Thanks I'll follow your installation steps then. I expect I'll learn from it. Its not too obvious what a snap actually is. Ubuntu seemed to introduce the term without giving an overview of what they actually do. Installing stuff is definitely the hardest part of using Linux, from my ignorant experience. Thanks for your time anyway.
